src/components/genericPopup/GenericPopup.js

```diff
@@ -0,0 +1,41 @@
+import styles from "@/styles/genericPopup.module.scss";
+
+const GenericPopup = ({
+  popupTitle,
+  popupMessage,
+  setDisplayPopup,
+  setShowLogInModal
+}) => {
+  const handleClose = () => {
+    setDisplayPopup(false);
+    if (
+      popupTitle.includes("Nu är du inloggad") ||
+      popupTitle.includes("Du bokar biljetter med e-posta")
+    ) {
+      setShowLogInModal(false);
+    } else if (popupTitle.includes("Din session har gått")) {
+      setShowLogInModal(true);
+    }
+  };
+
+  let smiley = false;
+  if (popupTitle.includes("Tack för denna gång")) {
+    smiley = true;
+  }
+
```

Comment:
Perhaps you could use a ternary operator for a bit cleaner code something like:
let smiley = popupTitle.includes("Tack för denna gång") ? "✌️" : null;

Reply:
nice tip. Didnt think about that. Thanks :)